spring-cloud-task-docs/src/main/asciidoc/features.adoc

```diff
@@ -52,7 +52,7 @@ updated in the repository with the results.
 
 NOTE: If the application requires the `ApplicationContext` to be closed at the
 completion of a task (all `*Runner#run` methods have been called and the task
-repository has been updated), set the property `spring.cloud.task.closecontext_enabled`
+repository has been updated), set the property `spring.cloud.task.closecontextEnabled`
```

Comment:
The name sounds ok to me. But, where is this implemented? Shouldn't that change there as well?

Reply:
In this case the property already has the following format `closecontextEnabled`.   i.e. https://github.com/spring-cloud/spring-cloud-task/blob/master/spring-cloud-task-core/src/main/java/org/springframework/cloud/task/configuration/TaskProperties.java#L68
I guess we can either use camel case or kabob case.